### PR TITLE
Database config doc tweaks

### PR DESCRIPTION
**Pull request by richleland**
Updated the database config section to let users know about the foreman-\* packages available for each database type. Also updated the README to note the --auto flag to the jekyll command.

db-docs-update

### CODE REVIEW

**Comment by ohadlevy** [comment]
@richleland awesome ! thanks

File: _includes/manuals/1.1/3.5.3_database_setup.md [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+production:
+  adapter: sqlite3
+  database: db/production.sqlite3
+  pool: 5
+  timeout: 5000
+{% endhighlight %}
 
 #### MySQL
 
+<div class="alert alert-info">When using MySQL, you can install the foreman-mysql package. See <a href="#3.3InstallFromPackages">3.3 Install From Packages</a>.</div>
```
Comment (domcleal) [inline _includes/manuals/1.1/3.5.3_database_setup.md:24]: It'd be nice to imply it adds something, as it sort of implies to me that it replaces the configuration.  Perhaps, "When using MySQL, install the foreman-mysql package for support.  See ..."

File: _includes/manuals/1.1/3.5.3_database_setup.md [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+
+<pre>/etc/foreman/config/database.yml</pre>
+
+#### SQLite (default)
+
+<div class="alert alert-info">When using SQLite, you can install the foreman-sqlite package. See <a href="#3.3InstallFromPackages">3.3 Install From Packages</a>.</div>
+
+By default, the database will be created in the db subdirectory.
+
+{% highlight ruby %}
```
Comment (ohadlevy) [inline _includes/manuals/1.1/3.5.3_database_setup.md:14]: this is actually YAML, I wonder if highlight works with yaml syntax too?

Comment (richleland) [inline _includes/manuals/1.1/3.5.3_database_setup.md:14]: Yep YAML works, although it doesn't look much different! :smile: I've updated that and the blocks below to be bash where appropriate. Will submit in another PR.

File: _includes/manuals/1.1/3.5.3_database_setup.md (hunk 1 above)
Comment (richleland) [inline _includes/manuals/1.1/3.5.3_database_setup.md:24]: I wasn't sure if it's required or optional to use the "official" package. If it's required, what about "PostgreSQL support requires the foreman-postgresql package. See..."?

Comment (domcleal) [inline _includes/manuals/1.1/3.5.3_database_setup.md:24]: True, it's only really optional when running from source and I'd expect those users to have found out how it works there by now (separate bundler.d files and groups).
